File: Julia/new_approach/new_approach.jl
function reversenumber(number::String)::String
	len = length(number)
	bound = convert(Int64, floor(len / 2))

	arr = split(number, "")

	for i = 1:bound
		arr[i], arr[len - i + 1] = arr[len - i + 1], arr[i]
	end

	return join(arr, "")
end

number = readline()
res = reversenumber(number)
println(res)

File: Nim/new_approach/new_approach.nim
import strutils

proc reverseNumber(number : string) : string =
  let length = len(number)
  let bound = int(length / 2) - 1
  var arr = @number

  for i in 0..bound:
    swap(arr[i], arr[length - 1 - i])

  result = arr.join("")

let number : string = readLine(stdin)
let res : string = reverse_number(number)
echo res